internal/server/event_handlers.go

```diff
@@ -256,13 +256,15 @@ func (s *EventServer) handleEvent() func(w http.ResponseWriter, r *http.Request)
 
 			notification := *event.DeepCopy()
 			if alert.Spec.Summary != "" {
-				if notification.Metadata == nil {
-					notification.Metadata = map[string]string{
-						"summary": alert.Spec.Summary,
-					}
-				} else {
-					notification.Metadata["summary"] = alert.Spec.Summary
+				meta := notification.Metadata
+				if meta == nil {
+					meta = make(map[string]string)
+				}
+				for key, value := range alert.Spec.EventMetadata {
```

Comment:
This should happen even if summary is string empty

Reply:
I'm confused :thinking:

Reply:
oh, right!!! lemme fix it

Reply:
fixed :ok_hand:

Reply:
Thanks, looks good. We'll need to wait for GitHub to recover as now Actions are down.